TamperMonkey/BIT-乐学-修改侧边栏课程-GUI.user.js

```diff
@@ -1,32 +1,34 @@
 // ==UserScript==
 // @name         BIT-乐学-修改侧边栏课程-GUI
 // @namespace    http://tampermonkey.net/
-// @version      0.1.2
+// @version      1.0.0
 // @description  修改侧边栏显示的课程
 // @license      GPL-3.0-or-later
 // @supportURL   https://github.com/YDX-2147483647/BIT-enhanced/issues
 // @author       CJJ
 // @match        *://lexue.bit.edu.cn/*
+// @icon         https://lexue.bit.edu.cn/theme/image.php/eguru/theme/1724573654/favicon
 // @grant        GM_getValue
 // @grant        GM_setValue
 // @grant        GM_registerMenuCommand
+// @downloadURL https://update.greasyfork.org/scripts/488788/BIT-%E4%B9%90%E5%AD%A6-%E4%BF%AE%E6%94%B9%E4%BE%A7%E8%BE%B9%E6%A0%8F%E8%AF%BE%E7%A8%8B-GUI.user.js
+// @updateURL https://update.greasyfork.org/scripts/488788/BIT-%E4%B9%90%E5%AD%A6-%E4%BF%AE%E6%94%B9%E4%BE%A7%E8%BE%B9%E6%A0%8F%E8%AF%BE%E7%A8%8B-GUI.meta.js
 // ==/UserScript==
```

Comment:
```suggestion
// @grant        GM_setValue
// @grant        GM_registerMenuCommand
// ==/UserScript==
```

GreasyFork 会自己加链接，自己标了也会被改，所以不用标。

https://greasyfork.org/zh-CN/help/meta-keys#meta-updateURL